code/modules/uplink/uplink_item.dm

```diff
@@ -1313,6 +1313,13 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	limited_stock = 2 //you can't use more than two!
 	restricted_roles = list("Shaft Miner")
 
+/datum/uplink_item/role_restricted/traitorsec
+	name = "Oppressor Upgrade Module"
+	desc = "A box containing an cyborg upgrade module that instantly hacks a cyborg and unlocks the outlawed security module (in a inconspicuous black-and-red paintjob)"
```

Comment:
The module isn't explicitly outlawed, turning them off is a config setting. It seems like a weird continuity break to have the description say such if someone reenables them on their server.

https://github.com/tgstation/tgstation/blob/6401bb8264708d3fc55c22fd6f7ba261aebc11ee/code/controllers/configuration.dm#L142
https://github.com/tgstation/tgstation/blob/7567fbafc43fa6430f165e6c702e5db5907d5733/code/modules/mob/living/silicon/robot/robot.dm#L212
https://github.com/tgstation/tgstation/blob/dab51af41ef82c5f817fb0530a65398bf8910738/config/game_options.txt#L262

Reply:
Perhaps use text such as "Unlocks a hacked version of the Security module" instead?

Reply:
honestly if the config allows secborgs this module shouldn't be purchasable that's just going to be a waste of TC.

Reply:
Add a new check & continue [here](https://github.com/InsaneHyena/tgstation/blob/8851267ad8a821363eee857e96b0fe065d9f8510/code/modules/uplink/uplink_item.dm#L30) & add two new list vars, `enabled_config` & `disabled_config` on the uplink_item parent so you can assign certain config values to disable/enable certain bundles, similar to how include_modes works, imo

Then have the var on the actual datum be something like `enabled_config = list(config.forbid_secborg)` or something like that.

Reply:
Fine, but I'm not actually sure how to add the needed check.